buildpack.md

```diff
@@ -542,11 +551,19 @@ For each entry in `<plan>`:
 
 #### Bills-of-Materials
 
-When the build is complete, a BOM (Bill-of-Materials) describing the app image MAY be generated for auditing purposes.
-If generated, this BOM MUST contain all `bom` entries in each `launch.toml` at the end of each `/bin/build` execution, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
+Buildpacks MAY write standardized Bill of Materials (sBOM) files with extension `<ext>`, where `<ext>` MUST denote an sBOM media type based on Internet Assigned Numbers Authority (IANA) [assigned media types](https://www.iana.org/assignments/media-types/media-types.xhtml). The currently supported media types and their expected file extensions are as follows:
```

Comment:
It might be confusing to use the s in sbom to mean `standardized` instead of `software`.

Reply:
FWIW CycloneDx does make a distinction as it provides a bill of materials for non software packages as well like services and hardware.

Reply:
I'll change sBOM -> BOM everywhere.

Reply:
Please see dac2a68

Reply:
Following discussion in Core Team Sync, `a06a415` uses `SBOM` everywhere instead.